language/stdlib/modules/Offer.move

```diff
@@ -35,6 +35,96 @@ module Offer {
   public fun address_of<Offered>(offer_address: address): address acquires T {
     borrow_global<T<Offered>>(offer_address).for
   }
+
+  /**
+   **************** SPECIFICATIONS ****************
```

Comment:
This marker does not make sense in generated docs (see here: https://github.com/libra/libra/blob/ecb535bd60b0b2f7856ecbbcfc6f67356690efb7/language/stdlib/modules/doc/Offer.md). Move out of doc block?